Sources/Sentry/SentryCrashReportConverter.m

```diff
@@ -384,9 +384,15 @@ - (SentryDebugMeta *)debugMetaFromBinaryImageDictionary:(NSDictionary *)sourceIm
     if ([exceptionType isEqualToString:@"nsexception"]) {
         exception = [self parseNSException];
     } else if ([exceptionType isEqualToString:@"cpp_exception"]) {
-        exception =
-            [[SentryException alloc] initWithValue:self.exceptionContext[@"cpp_exception"][@"name"]
-                                              type:@"C++ Exception"];
+
+        NSString *cppExceptionName = self.exceptionContext[@"cpp_exception"][@"name"];
+        NSString *cppExceptionReason = self.exceptionContext[@"reason"];
+
+        NSString *exceptionValue =
+            [NSString stringWithFormat:@"%@: %@", cppExceptionName, cppExceptionReason];
+
+        exception = [[SentryException alloc] initWithValue:exceptionValue type:@"C++ Exception"];
```

Comment:
`l`: Do we consider the `type` as a human readable value or could it also be a machine-readable one like `cpp.exception`?

Reply:
I think this is the string that winds up showing in the dashboard like `EXC_BAD_ACCESS` here: 
<img width="434" alt="image" src="https://github.com/user-attachments/assets/ee45268b-f981-4e34-b671-bb8067ca7e09" />